### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by AnnaAAA3432**

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Make sure to deploy your page and provide a link in the PR description

**Comment by AnnaAAA3432** [comment]
- [DEMO LINK](https://AnnaAAA3432.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://AnnaAAA3432.github.io/layout_search-bar-airbnb/report/html_report/)

File: src/style.css [hunk 1]
```diff
@@ -1 +1,84 @@
...
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: AvenirBook;
+  src: url(../src/fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+}
+
+input{
```
Comment (l4st1m0za) [inline src/style.css:13]: It is not recommended to apply styles directly to the HTML tags. It's better to use classes for styling.

File: src/style.css [hunk 2]
```diff
@@ -1 +1,84 @@
...
+  font-size: 14px;
+}
+
+.input::placeholder {
+  font-weight: 300;
+  color: inherit;
+}
+
+.input_one input {
+  /* padding: 24px 0 23px 62px; */
```
Comment (l4st1m0za) [inline src/style.css:30]: It's recommended to remove unused or commented code to keep the codebase clean.

File: src/style.css [hunk 3]
```diff
@@ -1 +1,84 @@
...
+  font-family: AvenirBook, sans-serif;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.input_two, .input_one {
```
Comment (l4st1m0za) [inline src/style.css:36]: The classes .input_two and .input_one are applied the same style. Consider merging them into one class to reduce code duplication.

File: src/style.css [hunk 4]
```diff
@@ -1 +1,84 @@
...
+  margin-top: 20px;
+  position: relative;
+}
+
+.input_two input{
+  /* padding: 12px 0 8px 33px; */
```
Comment (l4st1m0za) [inline src/style.css:42]: It's recommended to remove unused or commented code to keep the codebase clean.

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
Good job, but you need to change some things

File: src/style.css (hunk 3 above)
Comment (maxim2310) [reply src/style.css:30]: not fixed

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good work, just fix previous comment and also now you have broken Demo link 
image

File: src/style.css [hunk 5]
```diff
@@ -1 +1,79 @@
...
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+
+.custom-input::before {
+  content: '';
+  background-size: contain;
+  background-image: url(./images/Search.svg);
```
Comment (loralevitska) [inline src/style.css:50]: Combine with `background` property

File: src/index.html [hunk 6]
```diff
@@ -11,6 +11,18 @@
...
+      <input class="input"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+        type="search">
+    </form>
+
+    <form class="custom-input" data-qa="small">
+      <input class="input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        type="search">
```
Comment (loralevitska) [inline src/index.html:24]:
```suggestion
      <input 
        class="input"
        placeholder="Try &quot;Los Angeles&quot;"
        type="search"
      >
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Good job!
Check my comments to improve

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
the code looks good, but where is your images?
image

File: src/index.html (hunk 6 above)
Comment (anastasiiavorobiova) [reply src/index.html:24]: Not fixed

File: src/index.html [hunk 7]
```diff
@@ -11,6 +11,18 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="custom-input" data-qa="big">
+      <input class="input"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+        type="search">
```
Comment (anastasiiavorobiova) [inline src/index.html:18]:
```suggestion
      <input 
        class="input"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
        type="search"
      >
```

**Review by anastasiiavorobiova — APPROVED** [review]
Great work! Consider fixing indents

**Pull request closed without merging**